ach/src/main/java/com/adyen/checkout/ach/internal/ui/model/ACHDirectDebitComponentParamsMapper.kt

```diff
@@ -53,29 +53,16 @@ internal class ACHDirectDebitComponentParamsMapper(
             is ACHDirectDebitAddressConfiguration.None -> {
                 AddressParams.None
             }
+
             is ACHDirectDebitAddressConfiguration.FullAddress -> {
                 AddressParams.FullAddress(
                     supportedCountryCodes = supportedCountryCodes,
-                    addressFieldPolicy = AddressFieldPolicyParams.Required
+                    addressFieldPolicy = AddressFieldPolicyParams.Required,
                 )
             }
         }
     }
 
-    private fun ACHDirectDebitComponentParams.override(
```

Comment:
If someone manually set a configuration for a certain component inside drop-in but with conflicting fields (different amount or shopper locale or client key etc), will the component still take the drop-in configuration as the source of truth?

Reply:
The values from `CheckoutConfiguration` will always be used no matter what you configure in drop-in or for a component. Only sessions is able to override these values